pkg/crc/preflight/preflight_linux.go

```diff
@@ -118,9 +93,29 @@ func getPreflightChecks() []Check {
 	checks = append(checks, nonWinPreflightChecks[:]...)
 	checks = append(checks, libvirtPreflightChecks[:]...)
 
+	distro, _ := getLinuxDistroInfo()
+	switch distro {
+	case "fedora", "centos", "rhel":
+		checks = append(checks, redhatPreflightChecks[:]...)
+	default:
+		logging.Warnf("dnsmasq related changes are not implemented for %s", distro)
```

Comment:
I would not mention "dnsmasq" here, just `"distribution-specific preflight checks are not implemented for %s"`, maybe just as a debug, and `"%s unsupported", distro`.
If you don't handle the error from `getLinuxDistroInfo()`, you can get a "" distro, so the message printing distro will be weird.